docs(en-US): references -> user profile -> basic and custom data (#98)
logto-io/docs: docs/docs/references/users/README.md

- [**Basic data**](#basic-data): contains basic info in pre-defined attributes, such as user id, username, email, phone number, and when the user last signed in.
- [**Custom data**](#custom-data): contains additional info in customizable attributes, such as user-preferred color and language.

> - [**Identities**](#identities): contains the user info retrieved from social sign-in (a.k.a. sign-in with a social connector), such as Facebook, GitHub, and WeChat.

Author: ```suggestion
- [**Identities**](#identities): contains the user info retrieved from social sign-in (i.e., sign-in with a social connector), such as Facebook, GitHub, and WeChat.
```

#### User ID

_User ID_ is a unique auto-generated key to identify the user in Logto.

#### Username

_Username_ is used for sign-in with _username_ and password.

When the user has not registered with their username, their _username_ may be empty.
When the _username_ is non-empty, it should be at most 128 characters, only contain letters, numbers, and underscores (`_`), and NOT start with a number.

#### Primary email

_Primary email_ is the user's email address, used for sign-in with the email and passcode.

When the user has not registered with their email, their _primary email_ may be empty.
When the _primary email_ is non-empty, it should be at most 128 characters.

#### Primary phone

_Primary phone_ is the user's phone number, used for sign-in with the phone number and passcode from SMS.

When the user has not registered with their phone number, their _primary phone_ may be empty.
When the _primary phone_ is non-empty, it should contain numbers prefixed with the
[country calling code](https://en.wikipedia.org/wiki/List_of_country_calling_codes) (excluding the plus sign `+`).

#### Name

_Name_ is the user's full name.

> It should be at most 128 characters.

Author: ```suggestion
Its max length is 128.
```

#### Avatar

_Avatar_ is the URL pointing to the user's avatar image.

If the user registers with a social connector like Facebook and WeChat, their _avatar_ may be retrieved from the social user info.

> It should be at most 2048 characters.

Reviewer: nit: How about saying "the max-length is xxx" instead of "it should be"

Author: Updated to 
```suggestion
Its max length is 2048.
```

> It should be at most 2048 characters.

Author: ```suggestion
Its max length is 2048.
```
